plugin/keyring_vault/vault_keys_list.cc

```diff
@@ -0,0 +1,44 @@
+#include "vault_keys_list.h"
```

Comment:
In keyring framework the design is to use plural - for instance keys_container. If I change it to singular now it will break the naming convention from keyring framework.